charts/stable/strapi/questions.yaml

```diff
@@ -28,7 +28,7 @@ questions:
           description: "Sets Node's Environment"
           schema:
             type: string
-            default: "production"
+            default: "development "
```

Comment:
nope on dev.

Reply:
That's the default for strapi, once you set it up you moved it to production.

Reply:
ill set back to production

Reply:
i found the env variable to set the tokens, pushing it now.